### PR TITLE
Adding work style

### PR DESCRIPTION
**Pull request by KamilDonda**
+ Adding work is mobile now
+ Dropzone is hiding in smaller width
+ After add work, user is redirected to his work page

### CODE REVIEW

**Review by Kwolik — APPROVED** [review]

**Review by JustD4nTe — APPROVED** [review]

File: web-client/src/components/DropZone.jsx [hunk 1]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
       margin: "0 auto",
       paddingTop: "30px",
+      color: "#6069A9",
+      "@media only screen and (max-width: 1480px)": {
+        display: "none",
+      },
+    },
+    dragText: {
+      color: "black",
+      fontSize: 24,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      fontSize: "24px",
```

File: web-client/src/components/DropZone.jsx [hunk 2]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
     },
     message: {
-      color: "red",
-      padding: "2%",
-      fontSize: "15px",
+      color: "#AD1457",
+      height: 19,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      height: "19px",
```

File: web-client/src/components/DropZone.jsx [hunk 3]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      fontSize: 16,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      fontSize: "16px",
```

File: web-client/src/components/DropZone.jsx [hunk 4]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      display: "block",
+      "@media only screen and (max-width: 1480px)": {
+        display: "none",
+      },
+    },
+    messageButton: {
+      color: "#AD1457",
+      height: 19,
+      fontSize: 16,
+      letterSpacing: "0px",
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      letterSpacing: "0",
```

File: web-client/src/components/DropZone.jsx [hunk 5]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      width: "400px",
+      lineHeight: "1.4em",
+      display: "block",
+      "@media only screen and (max-width: 1480px)": {
+        display: "none",
+      },
+    },
+    messageButton: {
+      color: "#AD1457",
+      height: 19,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      height: "19px",
```

File: web-client/src/components/DropZone.jsx [hunk 6]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      fontSize: 24,
     },
     message: {
-      color: "red",
-      padding: "2%",
-      fontSize: "15px",
+      color: "#AD1457",
+      height: 19,
+      fontSize: 16,
+      letterSpacing: "0px",
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      letterSpacing: "0",
```

File: web-client/src/components/DropZone.jsx [hunk 7]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      lineHeight: "1.4em",
+      display: "block",
+      "@media only screen and (max-width: 1480px)": {
+        display: "none",
+      },
+    },
+    messageButton: {
+      color: "#AD1457",
+      height: 19,
+      fontSize: 16,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      fontSize: "16px",
```

File: web-client/src/components/DropZone.jsx [hunk 8]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
     },
     inputInfo: {
-      height: "10%",
+      height: "19px",
       width: "100%",
       textAlign: "left",
-      paddingLeft: "5%",
-      color: "#aaaaaa",
+      paddingLeft: "24px",
+      letterSpacing: "0px",
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      letterSpacing: "0",
```

File: web-client/src/components/DropZone.jsx [hunk 9]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      color: "#767676",
+      fontSize: 16,
+      "@media only screen and (max-width: 1480px)": {
+        marginTop: "-80px",
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx:91]: :((((((

File: web-client/src/components/DropZone.jsx [hunk 10]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
-      height: "10%",
+      height: "19px",
       width: "100%",
       textAlign: "left",
-      paddingLeft: "5%",
-      color: "#aaaaaa",
+      paddingLeft: "24px",
+      letterSpacing: "0px",
+      color: "#767676",
+      fontSize: 16,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      fontSize: "16px",
```

File: web-client/src/components/DropZone.jsx [hunk 11]
```diff
@@ -27,54 +39,107 @@ export default function DropZone(props) {
...
+      fontSize: 21,
+      color: "#AD1457",
       cursor: "pointer",
     },
     dialog: {
       display: "none",
     },
+    btn: {
+      color: "#54457F",
+      size: 16,
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx]:
```suggestion
      size: "16px",
```

File: web-client/src/components/DropZone.jsx [hunk 12]
```diff
@@ -213,14 +277,15 @@ export default function DropZone(props) {
...
       </div>
-      <div style={divStyle}>
-        <img className={style.logo} src={pdfIcon} alt="Logo" />
+      <div style={divStyle} className={style.bottom}>
+        <PictureAsPdfOutlinedIcon className={style.logo} />
         <span className={style.fileName}>{fileName}</span>
         <span className={style.fileSize}>{fileSize}</span>
         <span className={style.exit} onClick={Delete}>
-          x
+          🗙
```
Comment (upsu90) [inline web-client/src/components/DropZone.jsx:287]: Can you use icon from material UI icons?

**Review by upsu90 — CHANGES_REQUESTED** [review]

**Comment by upsu90** [comment]
image
please add alignItems: "center" and marginTop: "16px"

**Review by upsu90 — APPROVED** [review]

**Pull request merged**